Cargo.toml

```diff
@@ -137,6 +137,9 @@ bytemuck = "1.7"
 # Needed to poll Task examples
 futures-lite = "1.11.3"
 crossbeam-channel = "0.5.0"
+postcard = { version = "1.0", features = ["alloc"] }
+bincode = "1.3"
+rmp-serde = "1.1"
```

Comment:
Wondering if we should have a dedicated test binary for these kinds of things. Definitely makes compiling examples longer by doing this.